source/opcode.cpp

```diff
@@ -650,6 +650,19 @@ bool spvOpcodeIsCommutativeBinaryOperator(SpvOp opcode) {
   }
 }
 
+bool spvOpcodeIsLinearAlgebra(SpvOp opcode) {
+  // TODO: implement the cases OpMatrixTimesScalar, OpVectorTimesMatrix,
+  // OpMatrixTimesVector, OpMatrixTimesMatrix and OpOuterProduct in the
+  // TransformationReplaceLinearAlgebraInstruction class.
```

Comment:
We don't really want this part of the code base, which has nothing to do with spirv-fuzz, make reference to spirv-fuzz-specific stuff.

I suggest you make this function complete - fill out the switch cases.  And then in the transformation add some code that restricts to the cases that we support.

Reply:
Done.